### ISSUE TITLE
v1 release 

### ISSUE THREAD

**Issue by TheTallBlonde1** [stspec/pyspssio#3]
I love the motivation to create this, have a SPSS loader for my company that has to use pyreadstat (for speed) and SavReaderWriter (for completeness).

What is the plan to get this package to version 1? Do you need help with testing (I am testing at the moment)? I can see that it has not been updated since September, what is the plan? Are you looking for contributors?

**Comment by stspec** [comment]
Yes. v1.0 should pass a run of automated tests (ex. using pytest) as part of the release process, which I haven't had the chance to setup yet. I'm more than happy to accept your help if you're able to contribute to that effort.
